src/main/java/io/confluent/connect/hdfs/partitioner/FieldPartitioner.java

```diff
@@ -32,48 +32,61 @@
 
 public class FieldPartitioner implements Partitioner {
   private static final Logger log = LoggerFactory.getLogger(FieldPartitioner.class);
-  private static String fieldName;
+  private static List<String> fieldNames;
```

Comment:
I am not familiar with this code yet, but it seems weird that this field is `static`

cc @ewencp